Library/Homebrew/brew.rb

```diff
@@ -104,6 +104,13 @@
     possible_tap = Tap.fetch(possible_tap.first) if possible_tap
 
     if !possible_tap || possible_tap.installed? || Tap.untapped_official_taps.include?(possible_tap.name)
+      blocked_tap = possible_tap && Tap.untapped_official_taps.include?(possible_tap.name)
```

Comment:
Can you move the `Tap.untapped_official_taps.include?(possible_tap.name)` into a variable so it's only checked once?